SDAPackage/SDA/drone.py

```diff
@@ -23,6 +23,10 @@ def set_drone_position(self, new_point):
         :param new_point: New point for the drone
         :type new_point: Numpy Array
         """
+        print('current point of drone')
+        print(self.point)
+        print('new point of drone')
+        print(new_point)
```

Comment:
All of these comments should be removed.

Reply:
done